### PR TITLE
collectd: don't presume /usr/local in plist

### PR DESCRIPTION
**Pull request by DomT4**
Re: https://github.com/Homebrew/homebrew/pull/37966#issuecomment-84656418

### CODE REVIEW

File: Library/Formula/collectd.rb [hunk 1]
```diff
@@ -10,8 +10,11 @@ class Collectd < Formula
   end
 
   # Will fail against Java 1.7
-  option "java", "Enable Java 1.6 support"
-  option "debug", "Enable debug support"
+  option "with-java", "Enable Java 1.6 support"
+  option "with-debug", "Enable debug support"
+
+  deprecated_option "java" => "with-java"
+  deprecated_option "debug" => "with-debug"
```
Comment (xu-cheng) [inline Library/Formula/collectd.rb]: Maybe `depends_on :java => ["1.6", :optional]`

Comment (DomT4) [inline Library/Formula/collectd.rb]: WFM. I didn't want to screw with the formula too much, but that makes sense.

Comment (mikemcquaid) [inline Library/Formula/collectd.rb]: :+1:

Comment (DomT4) [inline Library/Formula/collectd.rb]: Pushed.

**Comment by mikemcquaid** [comment]
Thanks again @DomT4!

**Pull request closed without merging**